test/models/NumericalDataSeries.spec.js

```diff
@@ -19,6 +19,18 @@ function (
             expect(function() { dataSeries.setData([1, 2, 3]); }).toThrow();
             expect(function() { dataSeries.setData([{x_value: '1', y_value: '2'}]); }).toThrow();
             expect(function() { dataSeries.setData([{x_value: 1, y_value: 2}]); }).not.toThrow();
+            expect(function() { dataSeries.setData([
+                {x_value: 1, y_value: 2},
+                {y_value: 1, x_value: 2}
+            ]); }).not.toThrow();
+            expect(function() { dataSeries.setData([
+                {x_value: 1, y_value: 2},
+                {y_value: 1, x_value: 2, z: 3}
+            ]); }).toThrow();
+            expect(function() { dataSeries.setData([
+                {y_value: 1, x_value: 2, z: 3},
+                {x_value: 1, y_value: 2}
+            ]); }).toThrow();
```

Comment:
Maybe add some tests to exercise/document the new fields (`x_min`, `x_max`, etc)? Or instead of using `z`, use a valid field so we can see that the object layout needs to be consistent.